### PR TITLE
Translate 2019 Accessibility chapter to Dutch

### PR DESCRIPTION
**Pull request by noah-vdv**
Makes progress on #1750

### CODE REVIEW

File: src/content/nl/2019/accessibility.md [hunk 1]
```diff
@@ -0,0 +1,320 @@
+---
+part_number: II
+chapter_number: 9
+title: Toegankelijkheid
+description: Toegankelijkheidshoofdstuk van de Web Almanac 2019 over leesgemak, media, navigatiegemak en compatibiliteit met ondersteunende technologieën.
+authors: [nektarios-paisios, obto, kleinab]
+reviewers: [ljme]
+analysts: [dougsillars, rviscomi, obto]
+translators: []
```
Comment (bazzadp) [inline src/content/nl/2019/accessibility.md:9]: You get credit for chapter translators! 😁

```suggestion
translators: [noah-vdv]
```

File: src/content/nl/2019/accessibility.md [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+kleinab_bio: Abigail Klein is een Google-software-engineer. Ze werkte aan de webtoegankelijkheid van Google Documenten, Spreadsheets en Presentaties, waar ze <a href="https://www.blog.google/outreach-initiatives/accessibility/whats-you-say-present-captions-google-slides/ toevoegde">automatische ondertiteling voor Google Presentaties</a>, evenals verbeterde ondersteuning voor schermlezer, braille, schermvergroting en hoog contrast. Ze werkt momenteel aan de toegankelijkheid van Google Chrome en ChromeOS. Ze heeft een bachelor en master in informatica van MIT, waar ze medeoprichter was van een ondersteunende technologie hackathon en labassistent en gastdocent was van de les ondersteunende technologie.
```
Comment (bazzadp) [inline src/content/nl/2019/accessibility.md:15]: What's the `toevoegde` all about?

```suggestion
kleinab_bio: Abigail Klein is een Google-software-engineer. Ze werkte aan de webtoegankelijkheid van Google Documenten, Spreadsheets en Presentaties, waar ze <a href="https://www.blog.google/outreach-initiatives/accessibility/whats-you-say-present-captions-google-slides/">automatische ondertiteling voor Google Presentaties</a>, evenals verbeterde ondersteuning voor schermlezer, braille, schermvergroting en hoog contrast. Ze werkt momenteel aan de toegankelijkheid van Google Chrome en ChromeOS. Ze heeft een bachelor en master in informatica van MIT, waar ze medeoprichter was van een ondersteunende technologie hackathon en labassistent en gastdocent was van de les ondersteunende technologie.
```

File: src/content/nl/2019/accessibility.md [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+### Hoofdinhoudsgebied
+
+Een [Hoofdinhoudsgebied <i lang="en">`main`</i>](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Main_role) geeft aan schermlezers aan waar de hoofdinhoud van een webpagina begint, zodat gebruikers er direct naartoe kunnen springen. Zonder dit moeten gebruikers van schermlezers uw navigatie handmatig overslaan elke keer dat ze naar een nieuwe pagina op uw site gaan. Dit is natuurlijk nogal frustrerend.
+
+We ontdekten dat slechts één op de vier pagina's (26,03%) een belangrijk Hoofdinhoudsgebied bevatte. En verrassend genoeg bevatte 8,06% van de pagina's ten onrechte meer dan één belangrijk Hoofdinhoudsgebied, waardoor deze gebruikers raden welk gebied de werkelijke hoofdinhoud bevat.
+
+{{ figure_markup(
+  image="fig4.png",
+  caption="Percentage pagina's op basis van hun aantal `main` elementen.",
+  description="Verticaal staafdiagram met percentagegegevens, variërend van 0 tot 80 in stappen van 20, versus balken die het aantal "Hoofd" inhoudsgebied per pagina van 0 tot 4 weergeven. Bron: HTTP Archive (juli 2019). Nul: 73,97%; Een: 17,97%; Twee: 7,41%; Drie: 0,15%; 4: 0,06%.",
```
Comment (bazzadp) [inline src/content/nl/2019/accessibility.md:151]: Can't have quotes within quotes and think this should be `main` anyway - could you correct this in English version and add to this PR?

```suggestion
  description="Verticaal staafdiagram met percentagegegevens, variërend van 0 tot 80 in stappen van 20, versus balken die het aantal `main` inhoudsgebied per pagina van 0 tot 4 weergeven. Bron: HTTP Archive (juli 2019). Nul: 73,97%; Een: 17,97%; Twee: 7,41%; Drie: 0,15%; 4: 0,06%.",
```

File: src/content/nl/2019/accessibility.md [hunk 4]
```diff
@@ -0,0 +1,320 @@
...
+
+Van de geanalyseerde pagina's specificeert 26,13% geen taal met het kenmerk `lang`. Hierdoor blijft meer dan een kwart van de pagina's vatbaar voor alle hierboven beschreven problemen. Het goede nieuws? Van de sites die het `lang`-attribuut gebruiken, specificeren ze 99,68% van de tijd een geldige taalcode correct.
+
+### Afleidende inhoud
+
+Sommige gebruikers, zoals mensen met cognitieve beperkingen, hebben moeite om zich gedurende lange tijd op dezelfde taak te concentreren. Deze gebruikers willen niet omgaan met pagina's met veel beweging en animaties, vooral als deze effecten puur cosmetisch zijn en geen verband houden met de taak die voorhanden is. Deze gebruikers hebben minimaal een manier nodig om alle afleidende animaties uit te schakelen.
+
+Helaas geven onze bevindingen aan dat oneindig herhalende animaties vrij gebruikelijk zijn op internet, waarbij 21,04% van de pagina's ze gebruikt via oneindige CSS-animaties of [`<marquee>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/marquee) en [`<blink>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blink) elementen.
```
Comment (bazzadp) [inline src/content/nl/2019/accessibility.md:98]: Can you check all MDN references to see if Dutch versions exist and use them if they do?

```suggestion
Helaas geven onze bevindingen aan dat oneindig herhalende animaties vrij gebruikelijk zijn op internet, waarbij 21,04% van de pagina's ze gebruikt via oneindige CSS-animaties of [`<marquee>`](https://developer.mozilla.org/nl/docs/Web/HTML/Element/marquee) en [`<blink>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blink) elementen.
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
Few things I've noticed

File: src/content/nl/2019/accessibility.md (hunk 3 above)
Comment (bazzadp) [reply src/content/nl/2019/accessibility.md:151]: Going to accept this one to allow the tests to run.

**Comment by rviscomi** [comment]
@noah-vdv thank you for your hard work translating content into Dutch! I love that we're making the Web Almanac more accessible to developers everywhere.

One thing I want to point out is that the 2019 content will be visited much less frequently than 2020 content. So I'd encourage you to prioritize 2020 chapters, especially the ones that are edited and getting the [most traffic](https://datastudio.google.com/u/0/reporting/1c89f8c3-7e11-4376-8815-39966e13180e/page/1M).

**Comment by noah-vdv** [comment]
> @noah-vdv thank you for your hard work translating content into Dutch! I love that we're making the Web Almanac more accessible to developers everywhere.
> 
> One thing I want to point out is that the 2019 content will be visited much less frequently than 2020 content. So I'd encourage you to prioritize 2020 chapters, especially the ones that are edited and getting the [most traffic](https://datastudio.google.com/u/0/reporting/1c89f8c3-7e11-4376-8815-39966e13180e/page/1M).

Np and will do 👌

File: src/content/nl/2019/accessibility.md (hunk 4 above)
Comment (noah-vdv) [reply src/content/nl/2019/accessibility.md:98]: Should I check all links for dutch versions?

**Comment by bazzadp** [comment]
On that note I’ve just merged in the final version of the 2020 Accessibility chapter if this is a topic of particular interest to you. If not grab the most popular one ready to go (currently Performance).

File: src/content/nl/2019/accessibility.md (hunk 4 above)
Comment (bazzadp) [reply src/content/nl/2019/accessibility.md:98]: Ideally yes. Not the most exciting part of the job this - we should find some way to automate this!

File: src/content/nl/2019/accessibility.md [hunk 5]
```diff
@@ -0,0 +1,320 @@
...
+
+Tabelbijschriften via het element [`<caption>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption) zijn nuttig om meer context te bieden aan alle soorten lezers. Een bijschrift kan een lezer voorbereiden om de informatie op te nemen die uw tabel deelt, en het kan vooral handig zijn voor mensen die snel afgeleid of onderbroken worden. Ze zijn ook handig voor mensen die hun plaats in een grote tabel kunnen verliezen, zoals een gebruiker van een schermlezer of iemand met een leer- of verstandelijke beperking. Hoe gemakkelijker u het voor lezers kunt maken om te begrijpen wat ze analyseren, hoe beter.
+
+Desondanks bevat slechts 4,32% van de pagina's met tabellen bijschriften.
+
+## Compatibiliteit met ondersteunende technologieën
+
+### Het gebruik van ARIA
+
+Een van de meest populaire en meest gebruikte specificaties voor toegankelijkheid op internet is de [Accessible Rich Internet Applications](https://www.w3.org/WAI/standards-guidelines/aria/) (ARIA) -standaard. Deze standaard biedt een groot aantal aanvullende HTML-attributen om het doel achter visuele elementen (d.w.z. hun semantische betekenis) over te brengen en tot wat voor soort acties ze in staat zijn.
```
Comment (bazzadp) [inline src/content/nl/2019/accessibility.md:239]:
```suggestion
Een van de meest populaire en meest gebruikte specificaties voor toegankelijkheid op internet is de <a href="https://www.w3.org/WAI/standards-guidelines/aria/" lang="en">Accessible Rich Internet Applications</a> (ARIA) -standaard. Deze standaard biedt een groot aantal aanvullende HTML-attributen om het doel achter visuele elementen (d.w.z. hun semantische betekenis) over te brengen en tot wat voor soort acties ze in staat zijn.
```

File: src/content/nl/2019/accessibility.md [hunk 6]
```diff
@@ -0,0 +1,320 @@
...
+
+#### Indicatoren van verplichte en ongeldige velden
+
+Als we een veld tegenkomen met een grote rode asterisk ernaast, weten we dat dit een verplicht veld is. Of als we op verzenden klikken en te horen krijgen dat er ongeldige invoer was, moet alles wat in een andere kleur is gemarkeerd, worden gecorrigeerd en vervolgens opnieuw worden ingediend. Mensen met een laag of geen zicht kunnen echter niet vertrouwen op deze visuele aanwijzingen, daarom zijn de HTML-invoerattributen `required`, `aria-required` en `aria-invalid` zo belangrijk. Ze bieden schermlezers het equivalent van rode asterisken en rood gemarkeerde velden. Als een leuke bonus, wanneer u browsers informeert welke velden vereist zijn, zullen ze [delen van uw formulieren valideren](https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/Form_validation) voor u. Geen Javascript nodig.
+
+Van de pagina's die formulieren gebruiken, gebruikt 21,73% `vereist` of `aria-vereist` bij het markeren van verplichte velden. Slechts één op de vijf sites maakt hiervan gebruik. Dit is een eenvoudige stap om uw site toegankelijk te maken en biedt alle gebruikers handige browserfuncties.
```
Comment (bazzadp) [inline src/content/nl/2019/accessibility.md:295]:
```suggestion
Van de pagina's die formulieren gebruiken, gebruikt 21,73% `required` of `aria-required` bij het markeren van verplichte velden. Slechts één op de vijf sites maakt hiervan gebruik. Dit is een eenvoudige stap om uw site toegankelijk te maken en biedt alle gebruikers handige browserfuncties.
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
Looks very good. Other than the MDN links (there's a lot in this chapter unfortunately) if you get a chance to see if NL versions, and a few things I spotted below, think this is good to merge.

**Comment by noah-vdv** [comment]
Alright, checked all the links and none (apart from the one you pointed out) had NL versions so if there's no other mistakes I think it's good to go.

**Comment by bazzadp** [comment]
> Alright, checked all the links and none (apart from the one you pointed out) had NL versions so if there's no other mistakes I think it's good to go.

D'oh! Sorry for wasting your time.

Merging now...

**Review by bazzadp — APPROVED** [review]

**Pull request merged**